datasets.toml

```
@@ -353,6 +353,30 @@ The exclusion criteria are:
   doi="10.5281/zenodo.6635964"
 
 
+[[datasets]]
+
+  key="de_Matteis_2024a"
+
+  [datasets.publication]
+  doi="10.1136/ard-2024-225853"
+eligibility_criteria = "To be included, patients with sJIA had to meet the International League of Associations for Rheumatology (ILAR) criteria15 or at least one of the earlier historical classification criteria (American College of Rheumatology (ACR) 1972 criteria16 or Durban's first revision of ILAR criteria17), or the Paediatric Rheumatology INternational Trials Organisation (PRINTO) criteria.18 Patients with AOSD had to meet Yamaguchi,19 Fautrel,20 or Medsger and Christy21 classification criteria. We also retained studies that used criteria for at least one of the above-mentioned classifications, but the final diagnosis was from the physician."
```

Should eligibility_criteria text be cleaned from references?
```suggestion
eligibility_criteria = "To be included, patients with sJIA had to meet the International League of Associations for Rheumatology (ILAR) criteria or at least one of the earlier historical classification criteria (American College of Rheumatology (ACR) 1972 criteria or Durban's first revision of ILAR criteria), or the Paediatric Rheumatology INternational Trials Organisation (PRINTO) criteria. Patients with AOSD had to meet Yamaguchi, Fautrel, or Medsger and Christy classification criteria. We also retained studies that used criteria for at least one of the above-mentioned classifications, but the final diagnosis was from the physician."
```